broadlink/remote.py

```diff
@@ -80,7 +80,10 @@ def _check_sensors(self, command: int) -> bytes:
         response = self.send_packet(0x6A, packet)
         check_error(response[0x22:0x24])
         payload = self.decrypt(response[0x38:])
-        return bytearray(payload[len(self._request_header) + 4 :])
+        data = payload[len(self._request_header) + 4 :]
+        if data[0] >= 0xc0:  # Firmware issue.
+            raise exception(-4026, "The device returned malformed data", payload)
+        return data
```

Comment:
Why did you remove `bytearray`?

Reply:
It is a legacy conversion from Python 2, which indexes bytes and bytearrays differently. We no longer support Python 2, so we don't need it.